python/graphstorm/gconstruct/utils.py

```diff
@@ -937,11 +937,15 @@ def partition_graph(g, node_data, edge_data, graph_name, num_partitions, output_
     if part_method is None:
         part_method = "None" if num_partitions == 1 else "metis"
 
+    balance_ntypes = {}
+    for ntype in node_data:
+        if "train_mask" in node_data[ntype]:
+            balance_ntypes[ntype] = th.tensor(node_data[ntype]["train_mask"])
```

Comment:
Can you create a issue about this? For GSProcessing pipeline we also need to do so for metis partition.